ppa/archive/gale.py

```diff
@@ -160,23 +160,57 @@ def _make_request(
             requests.codes.unauthorized,
             requests.codes.server_error,
         ]:
+            # Extract item ID from the URL (format: .../v1/item/GALE%7C{item_id})
+            item_id = None
+            if "GALE%7C" in rqst_url:
+                item_id = rqst_url.split("GALE%7C")[-1]
+
```

Comment:
Ah, I forgot that we don't have an item id in the context where we're making the actual request. I don't think we should have this logic here; let's drop this and remove the item id from the logging. The URL is enough to see what's going on.

Sorry for misleading you.